README.md

```diff
@@ -1,15 +1,33 @@
-# C4V-Py
+# c4v-py
+
+<p align="center">
+  <img width="125" src="assets/logo.png">
```

Comment:
image doesn't load for me :/

Reply:
Hmmm, how did you test this? I was using Visual Studio Code default Markdown preview tool on Mac OS.

Reply:
Just with github :) 
<img width="1974" alt="Screenshot 2021-04-18 at 18 03 16" src="https://user-images.githubusercontent.com/19265860/115153956-666af900-a070-11eb-8469-7a22fda3040e.png">

Reply:
The reason why it does not work is that the Github code preview requests the asset from the `master` branch. But, this has not been merged into `master` yet. Thus, it cannot load the image.

I wouldn't like to put a specific URL (something that includes the commit number) because then changing the logo becomes cumbersome.

Do you have alternative ideas?